.dockerfile

```diff
@@ -41,14 +34,22 @@ RUN --mount=type=cache,target=/home/hemlock/.opam/download-cache,uid=1000,gid=10
     && opam install -y --deps-only . \
     && rm Hemlock.opam
 
+FROM --platform=${HEMLOCK_PLATFORM} prod AS pre-push
+ARG HEMLOCK_PRE_PUSH_CLONE_PATH
+USER hemlock
+WORKDIR /home/hemlock/origin
+COPY --chown=hemlock:hemlock ${HEMLOCK_PRE_PUSH_CLONE_PATH}/ .
```

Comment:
The `gh-push` plugin wasn't properly setting this argument. Since it was an empty string, this line copied `/` to `.`. Since `/` is the base of our Hemlock repo, this set us up in a valid state for the `git clone` on line 43. The correct stuff got tested, but this was broken. I should have enforced that the arg be set.